.eslintrc.cjs

```diff
@@ -20,7 +20,17 @@ module.exports = {
         fixMixedExportsWithInlineTypeSpecifier: false,
       },
     ],
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
This one always annoyed me, since this was a library where there were going to be some explicit anys.